src/ProductConstructionService/ProductConstructionService.Api/PcsStartup.cs

```diff
@@ -284,4 +291,24 @@ public static bool IsGet(this HttpContext context)
     {
         return string.Equals(context.Request.Method, "get", StringComparison.OrdinalIgnoreCase);
     }
+
+    public static async Task ConfigureRedis(WebApplicationBuilder builder, bool isDevelopment, string? managedIdentityId)
+    {
+        var redisConfig = ConfigurationOptions.Parse(
+            builder.Configuration.GetConnectionString("redis") ?? throw new Exception("Missing Redis connection string."));
```

Comment:
we have an extension for this `builder.Configuration.GetRequiredValue` I think. Also please extract the key as a const

Reply:
you'll have to use `ConnectionStrings:redis` for the key tho